### ISSUE TITLE
Progressbar component should have an aria-label or aria-labelledby attribute

### ISSUE THREAD

**Issue by sarvanhd** [valor-software/ng2-bootstrap#6475]
The Progressbar component should have an input field for aria-label or aria-labelledby attribute for Bar component
